playgame.py

```diff
@@ -98,9 +121,9 @@ def parse_args():
                         action = 'store_true',
                         help = 'log moves and game state as game is played')
     parser.add_argument('-l', '--log_dir', dest = 'log_dir', default = None, 
-                        help = 'save logs to file')
+                        help = 'save logs to file (results/PATH_PROVIDED)')
     parser.add_argument('-e', '--log_stderr', dest = 'log_stderr',
-                        help = 'log errors to file')
+                        help = 'log errors to file (results/PATH_PROVIDED)')
```

Comment:
since we're gonna support arbitrary paths, please undo this change. Instead, update sample.sh and sample_tournament.sh to log to a results directory explicitly